provider/resource_keycloak_realm.go

```diff
@@ -73,6 +73,70 @@ func resourceKeycloakRealm() *schema.Resource {
 				Computed: true,
 			},
 
+			//Smtp server
+
+			"smtp_server": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"starttls": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"auth": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"port": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"host": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"reply_to": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"reply_to_display_name": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"from": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"from_display_name": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"envelope_from": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"ssl": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"user": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"password": {
+							Type:      schema.TypeString,
+							Optional:  true,
+							Sensitive: true,
+							DiffSuppressFunc: func(_, remoteBindCredential, _ string, _ *schema.ResourceData) bool {
```

Comment:
Could you rename `remoteBindCredential`?  I think this was copied from the ldap user federation resource where this particular attribute was called `bind_credential`.

Reply:
ok